lib/usd/utils/tests/CMakeLists.txt

```diff
@@ -0,0 +1,64 @@
+find_package(GTest REQUIRED)
+
+set(USDUTILS_TEST_EXECUTABLE_NAME AL_USDUtilsTests)
+set(USDUTILS_TEST_NAME GTest:AL_USDUtilsTests)
+set(USDUTILS_PYTHON_TEST_NAME Python:AL_USDUtilsTests)
+
+add_definitions(
+  -DAL_EXTRAS_TEST_DATA="${CMAKE_CURRENT_SOURCE_DIR}/data"
+)
+
+list(APPEND usdutils_test_source
+    main.cpp
+    test_MayaTransformAPI.cpp
+)
+
+add_executable(${USDUTILS_TEST_EXECUTABLE_NAME}
+  ${usdutils_test_source}
+)
+
+target_link_libraries(${USDUTILS_TEST_EXECUTABLE_NAME}
+  PRIVATE
+    ${boost_python_LIBRARIES}
+    ${GTEST_LIBRARIES}
+    ${PYTHON_LIBRARIES}
+    arch
+    sdf
+    tf
+    gf
+    usd
+    usdGeom
+    vt
+    mayaUsdUtils
+    -lpthread -lm
+)
+
```

Comment:
I don't have access to my Linux machine for a while to test this but do we need to link against all of these libraries?

Instead of ${GTEST_LIBRARIES}, use GTest::GTest.  By doing this, you don't need to have ${GTEST_INCLUDE_DIRS} in the 
target_include_directories